server/index.js

```diff
@@ -2,9 +2,11 @@ const express = require('express');
 require('dotenv').config();
 const bodyParser = require('body-parser');
```

Comment:
I believe bodyParser is now built into Express and not needed as an explicit dependency in the server index.js

Reply:
good to know! that is helpful